src/App.tsx

```diff
@@ -42,6 +43,7 @@ export enum Routes {
   HOME = '/home',
   SETTINGS = '/settings',
   VOLUNTEER = '/volunteer',
+  TEAM = '/team',
```

Comment:
We'll want to set up the route so that it has a team ID parameter

Reply:
```suggestion
  TEAM = '/team/:id',
```
Agreed, this is the proper syntax I believe

Reply:
👍